pythonFiles/include/blender_vscode/load_addons.py

```diff
@@ -72,8 +76,12 @@ def create_link_in_user_addon_directory(directory, link_path):
         os.symlink(str(directory), str(link_path), target_is_directory=True)
 
 
-def is_in_any_addon_directory(module_path):
-    for path in addon_directories:
+def is_in_any_addon_directory(module_path: Path):
+    for repo in bpy.context.preferences.extensions.repos:
+        repo: bpy.types.UserExtensionRepo
+        if repo.enabled and repo.directory == module_path.parent:
+            return True
+    for path in ADDON_DIRECTORIES:
```

Comment:
This is the star of the show. I know, a bit too many changes, it is night as I am commiting, you need to forgive me.